### PR TITLE
Integrates the ROSA with HCP migration back into the main branch

### PR DESCRIPTION
**Pull request by EricPonvelle**
`enterprise-4.17+`

NOTE: **We are not squashing the commits in this PR because it's integrating an entire feature.**

### CODE REVIEW

File: snippets/ztp_PtpConfigDualCardGmWpc.yaml [hunk 1]
```diff
@@ -89,11 +89,11 @@ spec:
                 - "-p"
                 - "MON-HW"
               reportOutput: true
-            - args: #ubxtool -P 29.20 -p CFG-MSG,1,38,248
+            - args: #ubxtool -P 29.20 -p CFG-MSG,1,38,300
```
Comment (xJustin) [inline snippets/ztp_PtpConfigDualCardGmWpc.yaml:92]: Not sure why the formatting change on these snippets

File: snippets/ztp_PtpConfigGmWpc.yaml [hunk 2]
```diff
@@ -82,11 +82,11 @@ spec:
                 - "-p"
                 - "MON-HW"
               reportOutput: true
-            - args: #ubxtool -P 29.20 -p CFG-MSG,1,38,248
+            - args: #ubxtool -P 29.20 -p CFG-MSG,1,38,300
```
Comment (xJustin) [inline snippets/ztp_PtpConfigGmWpc.yaml:85]: Same formatting change

File: rest_api/objects/index.adoc [hunk 3]
```diff
@@ -1339,7 +1339,7 @@ Required::
 
 | `items`
 | xref:../oauth_apis/useroauthaccesstoken-oauth-openshift-io-v1.adoc#useroauthaccesstoken-oauth-openshift-io-v1[`array (UserOAuthAccessToken)`]
-|
+|
```
Comment (xJustin) [inline rest_api/objects/index.adoc:1342]: There are a lot of updates to this file and I am not so sure why the HCP migration would be involved here.

File: backup_and_restore/application_backup_and_restore/release-notes/oadp-1-4-release-notes.adoc [hunk 4]
```diff
@@ -28,4 +28,4 @@ include::modules/oadp-upgrading-oadp-operator-1-4-0.adoc[leveloffset=+3]
 
 To upgrade from OADP 1.3 to 1.4, no Data Protection Application (DPA) changes are required.
 
-include::modules/oadp-verifying-upgrade-1-4-0.adoc[leveloffset=+2]
\ No newline at end of file
+include::modules/oadp-verifying-upgrade-1-4-0.adoc[leveloffset=+2]
```
Comment (xJustin) [inline backup_and_restore/application_backup_and_restore/release-notes/oadp-1-4-release-notes.adoc:31]: Does this even change anything? Probably just some weird artifact.

File: _topic_maps/_topic_map_rosa_hcp.yml [hunk 5]
```diff
@@ -127,8 +101,6 @@ Distros: openshift-rosa-hcp
 Topics:
 - Name: Tutorials overview
   File: index
-# - Name: ROSA prerequisites
```
Comment (eohartman) [inline _topic_maps/_topic_map_rosa_hcp.yml:130]: Other tutorials are in here, so I'm not sure why this one is commented out.

File: installing/installing_openstack/installing-openstack-three-node.adoc [hunk 6]
```diff
@@ -13,4 +13,4 @@ You can install a three-node cluster on installer-provisioned infrastructure onl
 include::modules/installation-three-node-cluster-cloud-provider.adoc[leveloffset=+1]
 
 == Next steps
-* xref:../../installing/installing_openstack/installing-openstack-installer-custom.adoc#installing-openstack-installer-custom[Installing a cluster on OpenStack with customizations]
\ No newline at end of file
+* xref:../../installing/installing_openstack/installing-openstack-installer-custom.adoc#installing-openstack-installer-custom[Installing a cluster on OpenStack with customizations]
```
Comment (bhardesty) [inline installing/installing_openstack/installing-openstack-three-node.adoc:16]: I'm not sure why this is coming in, but it seems harmless.

File: rest_api/objects/index.adoc (hunk 3 above)
Comment (aireilly) [reply rest_api/objects/index.adoc:1342]: Yeah this is not right.

File: modules/telco-ran-yaml-ref-day-2-operators.adoc
Comment (aireilly) [inline modules/telco-ran-yaml-ref-day-2-operators.adoc:1]: This file should not be in the PR.

File: snippets/ztp_PtpConfigDualCardGmWpc.yaml
Comment (aireilly) [inline snippets/ztp_PtpConfigDualCardGmWpc.yaml:1]: This file should not be in the PR.

File: snippets/ztp_PtpConfigGmWpc.yaml
Comment (aireilly) [inline snippets/ztp_PtpConfigGmWpc.yaml:1]: This file should not be in the PR.

File: modules/rosa-policy-security-and-compliance.adoc [hunk 7]
```diff
@@ -118,8 +118,4 @@ to ensure application and data security controls are properly
 enforced.
 - Use IAM tools to apply the appropriate permissions to AWS
 resources in the customer account.
-|===
-
-.Additional resources
-
-* For more information about customer or shared responsibilities, see the xref:../../rosa_architecture/rosa_policy_service_definition/rosa-policy-process-security.adoc#rosa-policy-process-security[ROSA Security] document.
\ No newline at end of file
+|===
```
Comment (EricPonvelle) [inline modules/rosa-policy-security-and-compliance.adoc:121]: This x-ref in the module was breaking the build.

**Pull request merged**